src/components/PlayerDetail.js

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+
+export default function PlayerDetail({ player }) {
+  return (
+    <>
+      <h1>{player.name}</h1>
+      <div>
+        <p>position: {player.position}</p>
+        <p>jersey number: {player.id}</p>
+        <p>
+          team: <Link to={`/teams/${player.team_id}`}>{player.team_id}</Link>
```

Comment:
`>{player.teams.name}< would ` would show the team name inside link instead of the id